resources/js/components/EditProject.vue

```diff
@@ -17,7 +17,7 @@
       <a :href="$baseURL + '/project/' + cdash.project.Id + '/edit'">edit the project</a>
       <br>
       Click here to
-      <a :href="$baseURL + '/generateCTestConfig.php?projectid=' + cdash.project.Id">download the CTest configuration file</a>
+      <a :href="$baseURL + '/' + cdash.project.Id + '/ctest_configuration'">download the CTest configuration file</a>
```

Comment:
Both of these URLs lack the `project`  folder to properly hit the API: 

![image](https://github.com/Kitware/CDash/assets/542106/85a14746-ccf4-459b-8fd9-2ca841291c99)

Reply:
Good catch.  I tested by manually typing in the URL and forgot to check the links I changed.